nexus/db-model/src/inventory.rs

```diff
@@ -966,6 +1009,7 @@ impl InvSledConfigReconciler {
         boot_partition_a_error: Option<String>,
         boot_partition_b_error: Option<String>,
     ) -> Self {
+        // TODO-john replace this column with the hw_host_phase_1_slot enum?
```

Comment:
Is this "todo" meant to be part of this PR?

Reply:
Ahh, I meant for it to be, but then I started on it and it's more complicated than I expected. I'll file an issue and change this comment to point to the issue; thanks.

Reply:
Filed #8642 and updated this comment to reference it